qiskit/chemistry/ground_state_calculation/adapt_vqe.py

```diff
@@ -209,8 +215,11 @@ def compute_groundstate(self, driver: BaseDriver) -> 'AdaptVQEResult':
             logger.info("Final maximum gradient: %s", str(np.abs(max_grad[0])))
 
         # once finished evaluate auxiliary operators if any
-        if aux_operators is not None and aux_operators:
-            vqe.compute_minimum_eigenvalue(operator, aux_operators)
+        if aux_operators is not None:
+            aux_result = vqe.compute_minimum_eigenvalue(operator, list(aux_operators.values()))
```

Comment:
```suggestion
            aux_result = vqe.compute_minimum_eigenvalue(operator, aux_operators)
```
This is already a list now, right?